### ISSUE TITLE
bug: Cmd+J opens accessibility preference pane, regardless of focussed app

### ISSUE THREAD

**Issue by dschuessler** [janhq/jan#2483]
**Describe the bug**
On macOS, when Jan is opened and has not been given permission to control the computer, Cmd+J opens the accessibility preference pane, hence rendering the shortcut unusable in every other app.

**Steps to reproduce**
1. Make sure, Jan does not have permission to control the computer.
2. Open Jan.
3. Focus another app that uses Cmd+J as hotkey (for example Visual Studio Code).
5. Press Cmd+J.

**Actual behavior**
The focused app does not receive the Cmd+J hotkey. Instead, the accessibility preference pane is brought up.

**Expected behavior**
The focused app receives the Cmd+J hotkey and does its thing. (In the case of Visual Studio Code that would be toggling the panel visibility.)

**Environment details**
- Operating System: macOS 14.1
- Jan Version: 0.4.9
- Processor: Apple M2
- RAM: 24GB

**Additional context**
It appears from [the code](https://github.com/janhq/jan/blob/1ad794c8a75a1867a51e726026a0f170b6b8c736/electron/main.ts#L115) that Cmd+J is registered as global hotkey for a feature called "quick ask". Likely, global hotkeys require permission to control the computer, which would explain why macOS deems it appropriate to bring up the preference pane.

**Comment by Van-QA** [comment]
hi @dschuessler, the quick ask feature is still under experimental atm, for temporally workaround, you can turn it off in the setting for the time being 🙏 
![image](https://github.com/janhq/jan/assets/64197333/edf9870d-fac7-43c7-bd71-f6b5087da5da)

**Comment by Van-QA** [comment]
Potentially related to https://github.com/janhq/jan/issues/2589. Please follow up using that issue 🙏
